doc/source/user_guide/meshing_workflow/new_meshing_workflows.rst

```diff
@@ -0,0 +1,701 @@
+.. _ref_user_guide_new_meshing_workflows:
+
+Meshing workflow
+================
+You can use PyFluent to access the new, enhanced meshing workflows.
+
+Watertight geometry meshing workflow
+------------------------------------
+Use the **Watertight Geometry** workflow for water-tight CAD geometries that
+do not require much in the way of cleanup or modifications.
```

Comment:
"much in the way of" - aha, that come from the product doc, which we are reusing,

Reply:
Also, this is quite useful - from the web: _This is useful for clean CADs or clean geometries that have already been prepped in another software, such as Ansys SpaceClaim._